integration/docker-compose.spdk.yml

```diff
@@ -3,6 +3,7 @@ version: "3.7"
 services:
 
   spdk-target:
+    image: ghcr.io/opiproject/opi-spdk:main
     build:
```

Comment:
If we never want to build it from here, should we remove the build context?

Reply:
left context here for the case people can't access ghcr or want to change dockerfile locally